swiftshield-Sources/XcodeProjectBuilder.swift

```diff
@@ -25,13 +27,17 @@ final class XcodeProjectBuilder {
         Logger.log(.buildingProject)
         let path = "/usr/bin/xcodebuild"
         let projectParameter = isWorkspace ? "-workspace" : "-project"
-        let arguments: [String] = [projectParameter, projectToBuild, "-scheme", schemeToBuild]
+        var arguments: [String] = [projectParameter, projectToBuild, "-scheme", schemeToBuild]
+        if sdkMode {
+            arguments += ["-sdk", "iphoneos"]
+        }
```

Comment:
Hm, why does sdkMode involves different arguments? Is this a debug thing that you forgot to remove?

Reply:
This maybe is another issue. XCode 11 intruduces a new flag "Supports Mac Catalyst", it means you can compile your iPAD project for Mac. For App project, this flag default is NO, for Framework project, it is YES by default. If we don't want to add this, we need to document it at least.

Reply:
Hmm does it fail to build without it? I was under the impression that having a properly codesigned project wouldn't need these arguments.

Reply:
Yes, the build will fail because of this error, 
"xcodebuild: error: Failed to build project XXXX with scheme XXXX.
	Reason: The run destination My Mac is not valid for Running the scheme 'XXXX'."